cmd/erasure-healing.go

```diff
@@ -285,6 +285,29 @@ func (er erasureObjects) healObject(ctx context.Context, bucket string, object s
 	storageDisks := er.getDisks()
 	storageEndpoints := er.getEndpoints()
 
+	if globalTrace.NumSubscribers(madmin.TraceHealing) != 0 {
+		startTime := time.Now()
+		defer func() {
+			tr := madmin.TraceInfo{
+				TraceType: madmin.TraceHealing,
+				Time:      startTime,
+				NodeName:  globalLocalNodeName,
+				FuncName:  "heal.checkVersion",
+				Duration:  time.Since(startTime),
+				Message:   fmt.Sprintf("dry:%v, rm:%v, recreate:%v mode:%v", opts.DryRun, opts.Remove, opts.Recreate, opts.ScanMode),
+				Path:      fmt.Sprintf("%s/%s", bucket, object),
+			}
+			if versionID != "" && versionID != "null" {
+				tr.Path += " v-" + versionID
+			}
+			if err != nil {
+				tr.Error = err.Error()
+			} else {
+				tr.HealResult = &result
+			}
+			globalTrace.Publish(tr)
```

Comment:
Maybe let's move this to another function

Reply:
Let me give it a try. Not sure there is much to reuse.

Reply:
Works fine. Adding to `healBucket` as well.